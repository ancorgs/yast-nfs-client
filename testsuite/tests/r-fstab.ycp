/**
 * Module:
 *   NFS client configuration
 *
 * Summary:
 *   Routines testuite
 *
 * Authors:
 *   Martin Vidner <mvidner@suse.cz>
 *
 * $Id$
 */
{
    include "testsuite.ycp";
    include "nfs/routines.ycp";

    list nfs_entries = [
	$[
	    "file":"/home",
	    "freq":0,
	    "mntops":"defaults",
	    "passno":0,
	    "spec":"foo.bar.com:/home",
	    "vfstype":"nfs"
	 ],
	$[
	    "file":"/var/spool/mail",
	    "freq":0,
	    "mntops":"defaults",
	    "passno":0,
	    "spec":"foo.bar.com:/var/spool/mail",
	    "vfstype":"nfs"
	 ]
	];

    DUMP ("FstabTableItems");
    TEST (``(FstabTableItems (nfs_entries)), [], nil);
}
