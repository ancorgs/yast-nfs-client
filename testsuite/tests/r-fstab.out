Dump	FstabTableItems
Return	[`item (`id (0), "foo.bar.com ", "/home ", "/home ", "defaults "), `item (`id (1), "foo.bar.com ", "/var/spool/mail ", "/var/spool/mail ", "defaults ")]
